src/ui/zabapgit_js_common.w3mi.data.js

```diff
@@ -1841,3 +1841,151 @@ function enumerateJumpAllFiles() {
         title:  title
       };});
 }
+
+function DiffColumnSelection_Initialize() {
+  DiffColumnSelection.prototype.addMousedownEventListener = DiffColumnSelection_addMousedownEventListener;
+  DiffColumnSelection.prototype.mousedownEventListener = DiffColumnSelection_mousedownEventListener;
+  DiffColumnSelection.prototype.addCopyEventListener = DiffColumnSelection_addCopyEventListener;
+  DiffColumnSelection.prototype.copyEventListener = DiffColumnSelection_copyEventListener;
+  DiffColumnSelection.prototype.getSelectedText = DiffColumnSelection_getSelectedText;
+  var columnSelection = new DiffColumnSelection();
+  return columnSelection;
+}
+
+function DiffColumnSelection() {
+  this.addMousedownEventListener();
+  this.addCopyEventListener();
+  this.selectedColumnIdx = -1;
+  this.lineNumColumnIdx = -1;
+}
+
+function DiffColumnSelection_addMousedownEventListener() {
+  document.addEventListener("mousedown",
+    //https://stackoverflow.com/questions/2749244/javascript-setinterval-and-this-solution
+    (function(self) {
+      return function(e) {
+        self.mousedownEventListener(e);
+      };})(this));

```

Comment:
there is a `bind` polyfill at the top, so the code can simplified like this

```js
function DiffColumnSelection_addMousedownEventListener() {
  //https://stackoverflow.com/questions/2749244/javascript-setinterval-and-this-solution
  document.addEventListener("mousedown", this.mousedownEventListener.bind(this));
}
```

Reply:
all changes done as requested

Reply:
done